pages/api/turso-global.ts

```diff
@@ -1,34 +1,26 @@
 import { NextRequest as Request, NextResponse as Response } from "next/server";
-import { Client as LibsqlClient, createClient } from "@libsql/client/web";
+import { createClient } from "@libsql/client/web";
 
 export const config = {
   runtime: "edge",
 };
 
-const start = Date.now();
-const client = buildLibsqlClient();
-
-function buildLibsqlClient(): LibsqlClient {
-    const url = process.env.TURSO_DB_URL?.trim();
```

Comment:
I would keep the checks to get a meaningful errors when required env variables are missing.